_includes/tutorials/dynamic-output-topic/kstreams/markup/dev/run-producer.adoc

```diff
@@ -0,0 +1,22 @@
+////
+   Example content file for how to include a console produer(s) in the tutorial.
+   Usually you'll include a line referencing the script to run the console producer and also include some content
+   describing how to input data as shown below.
+
+   Again modify this file as you need for your tutorial, as this is just sample content.  You also may have more than one
+   console producer to run depending on how you structure your tutorial
+
+////
+
+In a new terminal, run:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/dynamic-output-topic/kstreams/code/tutorial-steps/dev/console-producer.sh %}</code></pre>
++++++
+
+When the console producer starts, it will log some messages and hang, waiting for your input. Each line represents input data for the Kafka Streams application.
+To send all of the events below, paste the following into the prompt and press enter:
```

Comment:
When I pasted all the lines, I got the following exception, which I did not get if I copy and pasted and sent one line at a time:
```

[I] ➜ docker exec -i schema-registry /usr/bin/kafka-avro-console-producer --topic input --broker-list broker:9092\
  --property "parse.key=true"\
  --property 'key.schema={"type":"long"}'\
  --property "key.separator=:"\
  --property value.schema="$(< src/main/avro/order.avsc)"
5:{"id":5,"name":"tp","quantity":10000, "sku":"QUA00000123"}
6:{"id":6,"name":"coffee","quantity":1000, "sku":"COF0003456"}
7:{"id":7,"name":"hand-sanitizer","quantity":6000, "sku":"QUA000022334"}
8:{"id":8,"name":"beer","quantity":4000, "sku":"BER88899222"}

kafka.common.KafkaException: No key found in line
        at io.confluent.kafka.formatter.AvroMessageReader.readMessage(AvroMessageReader.java:191)
        at kafka.tools.ConsoleProducer$.main(ConsoleProducer.scala:55)
        at kafka.tools.ConsoleProducer.main(ConsoleProducer.scala)

```

Reply:
@rspurgeon good catch, that was the case, and I've removed the return at the end of the file. I re-ran it locally, and I was able to copy->paste the entire block and didn't get any errors.